packages/shared/src/utils/helpers/duration.ts

```diff
@@ -120,10 +173,10 @@ export const hasIntradayDurationUnit = duration_units_list =>
  * @param {String} expiry_type
```

Comment:
Not sure about `duration_units_list` types in **hasIntradayDurationUnit** function!

Reply:
according to tests in durations.js, it's an array of objects with 2 string values 🙂:
`Record<text' | 'value', string>[]`